package.json

```diff
@@ -75,7 +75,11 @@
     "ext": "js,json,html,njk"
   },
   "lint-staged": {
-    "*.{ts,js,css}": [
+    "*.{js,css}": [
+      "prettier --write",
```

Comment:
This is so we don't mess up the layout of the DecisionTree file. It got to the point where the linter was no longer doing a good job

Reply:
Yeah but we should have it in - can we do exclusions? If not then i guess it is tech debt that we'll have to sort quickly